### PR TITLE
Update plugins..

### PR DESCRIPTION
**Pull request by dkhamsing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,6 +289,11 @@ Projects in Swift language will be marked with :large_orange_diamond: feel free
  * [Backlight-for-XCode](https://github.com/limejelly/Backlight-for-XCode) - Highlights the current editing line in Xcode
  * [UIColor-Hex-Swift](https://github.com/yeahdongcn/UIColor-Hex-Swift) - Convenience method for creating autoreleased color using RGBA hex string. :large_orange_diamond:
  * [KPRunEverywhereXcodePlugin](https://github.com/kitschpatrol/KPRunEverywhereXcodePlugin) - An Xcode plugin to build and run an app across multiple iOS devices with one click.
+ * [RevealPlugin](https://github.com/shjborage/Reveal-Plugin-for-Xcode) - Plugin for Xcode to integrate the Reveal App to your project automatic
+ * [RealmPlugin](http://realm.io/docs/cocoa/0.81.0/)- Xcode plugin to generate new Realm models
+ * [AdjustFontSize](https://github.com/zats/AdjustFontSize-Xcode-Plugin) - Instant font size adjustment with `⌘ +` / `⌘ -`
+ * [Lin](https://github.com/questbeat/Lin) - Xcode plugin that provides auto-completion for NSLocalizedString
+ * [Rephrase](https://www.rephrase.io) - Localise from Xcode
```
Comment (ceyhuno) [inline README.md]: Rephrase is not open source nor free

Comment (dkhamsing) [inline README.md]: Not everything on the list is free (HockeyApp)

Comment (vsouza) [inline README.md]: if the list item is not free, it needs to have a version that is at least. If you have any item that does not fit the requirements should be removed.

**Comment by dkhamsing** [comment]
Ok

File: README.md (hunk 1 above)
Comment (dkhamsing) [inline README.md]: Ok

Comment (dkhamsing) [inline README.md]: @vsouza 

The Rephrase plugin is free, existing translation is free. Rephrase just offers a translation service on top of the plugin. Translation is not really a free service in general.. except for crowdsourced versions.

`Lin` is an alternative plugin, it's also free but open-source. It does not offer translation service.

Anyway, let me know if you would like me to remove Rephrase, no biggie

Thx

**Comment by lfarah** [comment]
Can this be merged?

**Comment by dkhamsing** [comment]
Uh no. Hang tight

**Comment by dkhamsing** [comment]
Can be merged now

**Comment by vsouza** [comment]
merged :smile:

**Comment by dkhamsing** [comment]
thx

**Pull request merged**